app/views/posts/show.html.erb

```diff
@@ -1 +1,6 @@
-<h1>Post id is <%= @post %></h1>
\ No newline at end of file
+<%= render  @post	%>
+
+
+
+<%= render @post.comments	 %>
```

Comment:
Same here. Kindly display the post number along with the post details. I have provided the reference with the wireframe design. ⬇️

| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://github.com/MosDevx/BlogApp/assets/33268927/4d15ef1c-b8cf-4cbe-9fb6-ff0943340e25) | ![image](https://github.com/MosDevx/BlogApp/assets/33268927/8b075ded-24e7-4541-aaf6-2f909f0890ef) |

Reply:
done